### DISCUSSION TITLE
Adding a tutorial on how to import the Unity Plugin through the git URL instead of editing the manifest.json

### DISCUSSION THREAD

**Discussion by xCISACx** [consulo/consulo#430] [General] [answered]
I noticed that a person I know had some issues editing the manifest.json when attempting to install the Consulo plugin.
A regular user won't know how to properly edit the manifest.json and will most likely encounter errors, so I'd like to suggest adding a tutorial on how to install the package through the Unity Package Manager using the git URL.

Simply choosing Add Package From Git URL and pasting this URL (https://github.com/consulo/UnityEditorConsuloPlugin.git) into it will download the latest version of the plugin through the Unity Package Manager, without a need to editing the manifest.json.

![image](https://user-images.githubusercontent.com/37281623/99075048-b8cf5780-25b0-11eb-9711-fc2b58aa2759.png)

![image](https://user-images.githubusercontent.com/37281623/99075121-dc929d80-25b0-11eb-8884-aad18dd69c3f.png)

The manifest.json is automatically changed to add the required lines this way:

![image](https://user-images.githubusercontent.com/37281623/99075250-14014a00-25b1-11eb-9b86-85d3542e43cc.png)

I think this could greatly improve how easy it is to install the Unity plugin.

**Comment by VISTALL** [comment, marked as answer]
Good suggestion. Maybe i will add this as alternative to current install action, or to docs.

But using raw repository can be painful, since it's master, and I'm not use tags. Maybe after sometime i will use.

**Reply by xCISACx** [reply]
Yeah, the best course of action would probably be making a .unitypackage for the plugin, but I'm not sure if that would be possible since I don't know how they are made.

**Reply by VISTALL** [reply]
It's not good since - updating will be painful. Tags and/or UPM releases are best for that

**Reply by xCISACx** [reply]
I'll keep using the git method for now and we'll see if a better one comes up then :)
